docs/www/production-deployment.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Production Deployment
+order: 0
+---
+# Production Deployment
+
+This guide will take you through what is needed to setup a production cluster
+of Redpanda.
+
+## Requirements
+
+For the best performance the following are recommended:
+
+- XFS for the data directory of Redpanda (/var/lib/redpanda/data)
+- A kernel that is at least 3.10.0-514, but a 4.18 or newer kernel is preferred
+- Local NVMe
+- 2GB of memory per core
```

Comment:
i think missing raid-0 for larger machines.

Reply:
Added